### DISCUSSION TITLE
App on DEV is too lazy

### DISCUSSION THREAD

**Discussion by gleidsonbalcazar** [gokulkrishh/expense.fyi#96] [General]
Hi, firstly congratulations.

Why when I started the project in dev env, it is slowly? Each process request it takes too long.

**Comment by gokulkrishh** [comment]
@gleidsonbalcazar Thanks. I have the same issue too. Haven't spend time on why part of it. My guess most likely, optimisation of bundles are done after build and npm run start takes care of serving the file in prod. But in dev mode everything is just raw file.

**Reply by gleidsonbalcazar** [reply]
Good explained! I have spent hours searching best configs. Firstly I changed config tailwind, just only after I discovered then tailwind 3+ already have jit, after that I know it's not worth spending time on this. Thank you!
